backend/app/schemas/user.py

```diff
@@ -17,7 +17,7 @@ class Auth(SchemaBase):
 
 
 class AuthLogin(Auth):
-    captcha: str
+    captcha: str | None = None
```

Comment:
I believe that this is only temporary, would it be better if this code was commented out directly?